src/pages/articles/software-engineering/singleton/index.md

```diff
@@ -0,0 +1,45 @@
+---
+title: "Singleton"
+parent: "software-engineering"
```

Comment:
Meta information will be build automatically, all of this will be overwritten except for the title